buildSrc/private/src/main/kotlin/androidx/build/jetbrains/JetBrainsAndroidXImplPlugin.kt

```diff
@@ -187,12 +189,16 @@ private fun enableArtifactRedirectingPublishing(project: Project) {
     }
 }
 
+@OptIn(ExperimentalBCVApi::class)
 private fun enableBinaryCompatibilityValidator(project: Project) {
     val androidXExtension = project.extensions.findByType(AndroidXExtension::class.java)
         ?: throw Exception("You have applied AndroidXComposePlugin without AndroidXPlugin")
     project.afterEvaluate {
         if (androidXExtension.shouldPublish()) {
             project.apply(plugin = "org.jetbrains.kotlinx.binary-compatibility-validator")
+            project.extensions.getByType(ApiValidationExtension::class.java).apply {
+                klib.enabled = true
```

Comment:
We also need to dump all KLib iOS API with this library, and setup CI checks for changes